docs/software-platforms/aws/working-with-snapshots.md

```diff
@@ -0,0 +1,114 @@
+# Working with snapshots on Lightsail for Research
+
+## What is a snapshot
+
+Snapshots of both [virtual computers](./creating-vcs.md) and [disks](./working-with-volumes.md) can be created in Lightsail for Research (LSfR).
+You can think of these snapshots as back-ups of your virtual computer or disk.
+
+As with backing up your computer, it is recommended to save snapshots frequently so that no data is lost.
+We recommend doing this once a week.
+
+- Snapshots can be used to restore or create a new virtual computer or disk.
+This is particularly helpful if you need to restore a virtual computer or disk that has crashed for any reason.
+- When restoring a virtual computer from a snapshot, you can change the [instance size and select a different configuration for vCPUs and memory](./creating-vcs.md#choosing-an-instance).
```

Comment:
While reviewing I wondered if this linkedsection should be called "choosing an instance size" instead?